pkg/npm/npm.go

```diff
@@ -0,0 +1,63 @@
+package npm
+
+import (
+	"bytes"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"io"
+	"strings"
+)
+
+type NpmRegistryOptions struct {
+	DefaultNpmRegistry string
+	SapNpmRegistry     string
+}
+
+type runner interface {
+	SetEnv(e []string)
+	Stdout(out io.Writer)
+}
+
+type execRunner interface {
+	runner
+	RunExecutable(e string, p ...string) error
+}
```

Comment:
What is the benefit of splitting this into two interfaces?

Reply:
I don't know, did copy/paste from the "original".